webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+
+            <p><code>total</code> contains the total number of crash reports that matched the query. It is a positive integer.</p>
+
+            <p><code>facets</code> contains the results of the various aggregations that are set in the query. It is a JSON object containing ordered lists of JSON objects. Each sub-object represents a &quot;term&quot; and its count, and can have sub-aggregations. Theoratically, there can be any number of nested sub-aggregations.</p>
+
+            <h1 id="filters">Filters</h1>
+
+            <p>The first thing that Super Search allows you to do is to filter on our dataset to reduce it. For example, if you only want to see crashes that happened on a given product and version, or with a specific build id.</p>
+
+            <div class="example">
+                <p>Example</p>
+                <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?product=Firefox&amp;version=40.0</span></code></pre>
+            </div>
+
+            <p>There are lots of available filters, listed in the <a href="{{ url('documentation:supersearch_api') }}">API page</a>. That list is constantly changing as we add more data to our crash reports.</p>
+
+            <h2>Logic</h2>
+
+            <p>When using several filters together, you do not have full control over how filter are combined together. Note that all filters can receive several values. Here are the rules: </p>
+
+            <ul>
+                <li>if there are several filters with the same name:
+                    <ul>
+                        <li>if the operator is a range (see Operators section below), combine all range filters with an <code>AND</code></li>
+                        <li>combine all remaining filters with an <code>OR</code></li>
+                    </ul>
+                </li>
+                <li>filters are combined with an <code>AND</code> operator</li>
+            </ul>
+
+            <p>For example, <code>product=Firefox &amp; version=1.0 &amp; version=2.0 &amp; date=&gt;2000-01-01 &amp; date=&lt;2001-01-01</code> will translate to: </p>
+
+            <pre>( product = Firefox ) <b>AND</b>
+( version = 1.0 <b>OR</b> version = 2.0 ) <b>AND</b>
+( date &gt; 2000-01-01 <b>AND</b> date &lt; 2001-01-01 )</pre>
+
+            <h2>Operators</h2>
+
+            <p>Each filter supports a number of operators, that will allow you to refine your searches. Concretely, those operators translate to a string that will be put at the beginning of the value of the parameter. For example, <code>~</code> is the "contains" operator. If you want to search for all crashes that have a signature containing "moz", you would do this: </p>
```

Comment:
I think you should mention that the way operators work is that they're put right before the value. (And here mention the `~moz` example). But some exceptional operators (`__null__`) don't require a value and can be combined with the `!` operator. (And here show an example using something like `foo=~__true__`)